ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -847,15 +847,19 @@ const updateCopiedAbilities = function () {
         }
         update[skill.replace(' ', '_')] = ability;
       }
-      setAttrs(update);
+      setAttrs(update, () => {
+        log('updateArmorValue');
```

Comment:
You probably want to get rid of this logging now.

Reply:
Removed.